classes/controller/AdminController.php

```diff
@@ -3604,7 +3605,9 @@ protected function copyFromPost(&$object, $table)
                 }
```

Comment:
We aren't only hashing in the Back Office. Also the Front Office will need hashing.